build.gradle

```diff
@@ -44,11 +44,20 @@ subprojects {
     version '1.0.0-SNAPSHOT'
 
     afterEvaluate {
-        if (project.plugins.hasPlugin('application')
-                && project.plugins.hasPlugin('scala')) {
-            startScripts {
-                doLast {
-                    unixScript.text = configureUnixClasspath(unixScript)
+        if (project.plugins.hasPlugin('scala')) {
```

Comment:
In some recent project I am trying to avoid `afterEvaluate` as somehow it "feels" better to do things during normal setup phase. So instead of afterEvaluate pattern one can use `withPlugin`

```
    pluginManager.withPlugin('scala') {
               tasks.withType(ScalaCompile) {
            scalaCompileOptions.additionalParameters = gradle.scala.compileFlags
        }
    }
```

Just a suggestion. Either way is fine and works for us

Reply:
I just found out that this applies the actual plugin to all of the subproject. So that's not a suitable way to go about it for us I think.

Reply:
Simply dropping "afterEvaluate" doesn't help either. Some stuff still breaks. I'm going to leave it like it is now. We can optimize at a later stage I think.